test/reline/test_key_actor_vi.rb

```diff
@@ -697,6 +697,12 @@ def test_vi_delete_meta_with_arg
     assert_line_around_cursor('aaa bbb ', 'ccc')
     input_keys('2dl')
     assert_line_around_cursor('aaa bbb ', 'c')
+    input_keys('d2h')
+    assert_line_around_cursor('aaa bb', 'c')
+    input_keys('2d3h')
+    assert_line_around_cursor('aaa', 'c')
```

Comment:
When I perform the same action in Vim, I get a different result. Is this intentional?

1. Enter `aaa bbc`
2. Place the cursor on `c`
3. Enter `2d3h`
4. Only `c` remains

Reply:
Nice catch, looks like I misunderstood the correct behavior.
Updated the test to match with readline, fix implementation to use 2*3 = 6 as vi_arg in that case